crates/editor/src/editor.rs

```diff
@@ -1007,7 +1005,7 @@ struct CompletionsMenu {
     matches: Arc<[StringMatch]>,
     selected_item: usize,
     scroll_handle: UniformListScrollHandle,
-    selected_completion_resolve_debounce: Option<Arc<Mutex<DebouncedDelay>>>,
+    enable_completion_resolution: bool,
```

Comment:
```suggestion
    resolve_completions: bool,
```

NIT, some naming proposals.
Also, wondering: we disable this for assistant completions only, those aren't supposed to have any resolve provided anyway, are they? If true, we can remove the field entirely as it's not necessary.

Reply:
Good point, changed the name!